Automattic/wp-calypso
# Checkout: Add a partner logo during checkout when partner coupon is present

#### Changes proposed in this Pull Request

* Add partner logo at the checkout screen of Jetpack products with a partner coupon.
* Context 👉🏻  p1HpG7-dvB-p2#comment-51746
* Summarized objective of this PR 👉🏻  1201265858818217-as-1201439117106183/f

#### Testing instructions



* Accessing [Jetpack Backup Daily checkout](http://calypso.localhost:3000/checkout/jetpack/jetpack_backup_daily?coupon=COUPON_CODE_HERE) should be enough for testing this. Just make sure to have a coupon code to test, you can reach out to the Avalon team to get a testing coupon.
* No need for finishing the order.

Related to #59374

#### Screenshots
> Elements on this screen might slightly differ from what's on the latest accepted design, since the checkout experience was updated since the design for this was finalized.

![image](https://user-images.githubusercontent.com/5550190/150842650-16e3306d-ae70-4b03-9f94-8e3e4016a197.png)
